nipype/interfaces/base/core.py

```diff
@@ -276,6 +276,15 @@ def _check_version_requirements(self, trait_object, raise_exception=True):
             version = LooseVersion(str(self.version))
             for name in names:
                 min_ver = LooseVersion(str(trait_object.traits()[name].min_ver))
+                try:
+                    min_ver > version
+                except TypeError:
+                    iflogger.warning(
+                        'Nipype is having issues parsing the package version '
+                        'for Trait {} ({})'
+                        'You may want to check whehter {} is larger than {}'
+                        .format(name, self.__class__.__name__, version, min_ver)
+                        )
```

Comment:
We can use f-strings now that nipype is 3.6+. Also, you'll want to escape here, otherwise you'll run into the same problem on the following line.
```suggestion
                        'Nipype is having issues parsing the package version '
                        f'for Trait {name} ({self.__class__.__name__})'
                        f'You may want to check whether {version} is larger than {min_ver}'
                        )
                    return
```

Reply:
Just to add: We should use `continue`, not `return`. And we should do this for `max_ver`, as well, ~15 lines down.